zion/src/main/scala/edu/uci/ics/cloudberry/zion/common/Config.scala

```diff
@@ -12,6 +12,8 @@ class Config(config: Configuration) {
 
   val AwaitInitial = config.getString("neo.timeout.initial").map(parseTimePair).getOrElse(10 minutes)
 
+  val maxFrameLength = config.getString("neo.stream.max.frame.length").map(parseMemoryPair).getOrElse(8388608)
```

Comment:
rename to `MaxFrameLengthForNeoWS` to be more precise? 
the global static values deserves a explicit good name